contracts/merger/ExchangerTimelock.sol

```diff
@@ -0,0 +1,54 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+interface IExchanger {
+    function exchange(uint256 amount) external;
+}
+
+/** 
+ @title Send RGT straight to TRIBE timelock
+ @author Joey Santoro
+ @notice For Rari core contributors to trustlessly maintain incentive alignment
+*/
+contract ExchangerTimelock is Ownable {   
+    using SafeERC20 for IERC20;
+
+    IExchanger public immutable exchanger;
+    address public immutable timelock;
+
+    /// @notice guardian multisig
+    address public constant guardian = 0x8ace03Fc45139fDDba944c6A4082b604041d19FC;
```

Comment:
should note that this is the *rari* timelock, not the fei-guardian multisig

Reply:
changed address not comment, will fix